app/src/main/res/layout/activity_chat.xml

```diff
@@ -6,6 +6,13 @@
     android:layout_height="match_parent"
     tools:context=".database.ChatActivity">
 
+    <TextView
+        android:id="@+id/emptyTextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="No messages. Start chatting at the bottom!"
```

Comment:
@puf Hmmm, this is kind of an experiment so we'll have to see how it goes. I would recommend running `./gradlew check` to make sure lint and checkstyle are happy, but in this case it wants you to create and `@string` resource. You can just ignore it with `tools:ignore="HardcodedText"` if you want.